Add configuration parsing for IPv6 addresses

andresdb91 opened the pull request:
Uses standard format for IPv6: [ipv6]:port
Adds ipversion option [4|6]
Defaults to IPv4 if option is bad or not found (due to if/else)
Adds ipversion option to example configuration

andresdb91 committed:
Add configuration parsing for IPv6 addresses

Uses standard format for IPv6: [ipv6]:port
Adds ipversion option [4|6]
Defaults to IPv4 if option is bad or not found (due to if/else)
Adds ipversion option to example configuration

alexbakker reviewed (changes requested):
Thanks for bringing this to my attention! I propose addressing this a little differently. With something like ``data["address"].rsplit(":", maxsplit=1)`` we can parse IPv4 and IPv6 address/port in one line. That way, we wouldn't need the ``ipversion`` option you introduced.

andresdb91 committed:
Parse IPv6 without ipversion option

andresdb91 commented:
Great, it felt a bit hacky after all. I changed it and added a strip to take out the brackets, I also took out the ipversion option from the example config since it's not needed anymore.

alexbakker commented:
Thanks!
